src/interfaces/ProfileInterface.ts

```diff
@@ -0,0 +1,8 @@
+export interface ProfileInterface {
+  name: string;
+  role: string;
+  photo?: string;
+  github: string;
+  linkedin: string;
+  lattes: string;
+}
```

Comment:
I think it's best if everyone needs to have an avatar image and github, linkedin and lattes be optional, what you think?

```suggestion
export interface ProfileInterface {
  name: string;
  role: string;
  avatar: string;
  github?: string;
  linkedin?: string;
  lattes?: string;
}
```